.github/workflows/draft-release.yml

```diff
@@ -0,0 +1,199 @@
+        id: extract_branch
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      - name: Run testing for release-draft
+        if: ${{ github.event.inputs.draft-release == 'true' && steps.semantic-release-draft.outputs.TAG_NAME != '' }}
+        run: node .github/scripts/trigger-workflow.js
+        env:
+          WORKFLOW_NAME: '.github/workflows/cli-test-npm.yml'
+          REPO_NAME: twilio/twilio-cli
+          REPO_ACCESS_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+          BRANCH_NAME: ${{steps.extract_branch.outputs.branch}}
+          INPUTS: '{ "version": "${{ steps.semantic-release-draft.outputs.TAG_NAME }}" }'
+      - name: Update release
+        if: ${{steps.semantic-release-draft.outputs.TAG_NAME != ''}}
+        run: node .github/scripts/update-release.js
+        env:
+          TAG_NAME: ${{steps.semantic-release-draft.outputs.TAG_NAME}}
+          RELEASE_BODY: ${{github.event.inputs.change-log}}
+          REPO_NAME: twilio/twilio-cli
+          REPO_ACCESS_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          INPUTS: '{ "tag-name": "${{needs.release.outputs.tag-name}}" }'
+  oclif-release:
+    runs-on: ubuntu-latest
+    needs: [release]
+    steps:
+      - name: Checkout cli repo
+        uses: actions/checkout@v2
+      - run: |
+          git pull
+          make install
+      - name: Extract branch name
+        id: extract_branch
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      - name: Invoke Oclif workflow for draft release
+        if: ${{needs.release.outputs.draft-tag-name != '' && github.event.inputs.draft-release == 'true'}}
+        run: node .github/scripts/trigger-workflow.js
+        env:
+          WORKFLOW_NAME: 'oclif-release.yml'
+          REPO_NAME: twilio/twilio-cli
+          REPO_ACCESS_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+          BRANCH_NAME: ${{steps.extract_branch.outputs.branch}}
+          INPUTS: '{ "home-brew-branch": "${{github.event.inputs.homebrew-branch}}", "tag-name": "${{needs.release.outputs.draft-tag-name}}", "formula": "twiliodraft", "pre-release": "${{github.event.inputs.homebrew-prerelease}}"}'
+  oclif-draft-test:
+    runs-on: ubuntu-latest
+    needs: [oclif-release]
+    steps:
+      - name: Extract branch name
+        id: extract_branch
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      - name: Wait for draft release to succeed
+        uses: lewagon/wait-on-check-action@v1.0.0
+        with:
+          ref: ${{steps.extract_branch.outputs.branch}}
+          check-name: 'Oclif Release'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 10
+      - name: Run testing on release-draft
+        if: ${{ github.event.inputs.draft-release == 'true' }}
+        run: node .github/scripts/trigger-workflow.js
+        env:
+          WORKFLOW_NAME: '.github/workflows/cli-test-homebrew.yml'
```

Comment:
How are we ensuring homebrew is released before running homebrew tests?

Reply:
This action `lewagon/wait-on-check-action@v1.0.0` waits for the Oclif release to complete before triggering homebrew tests. 
 check-name: 'Oclif Release'

Reply:
But we should wait till hombrew workflow in homebrew repo right?

Reply:
That's right. There's this default wait interval `wait-interval: 10`, but since there's no way to wait on another repo's workflow, I'd try to verify if this works fine in case of delayed releases.